packages/shared/lib/features/interfaces/delegation-features.interface.ts

```diff
@@ -0,0 +1,5 @@
+import { IFeatureFlag } from './feature-flag.interface'
+
+export interface IDelegationFeatures extends IFeatureFlag {
+    delegationList?: IFeatureFlag
```

Comment:
Maybe the features of `Delegate output` and `Claim rewards` could be added. wdyt?